compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -249,8 +249,13 @@ object Parsers {
 
     /** Skip on error to next safe point.
      */
-    protected def skip(stopAtComma: Boolean): Unit =
+    protected def skip(): Unit =
```

Comment:
Smarter skip, though it is actually a little dumber because it now doesn't handle non-delimited comma-separated lists like `for x,y,z <- ...`